source/newcommitter.md

```diff
@@ -20,11 +20,46 @@ The [Contributor Ladder](https://community.apache.org/contributor-ladder.html) h
 
 {{% toc %}}
 
+<a name="NewCommitter-Summary"></a>
+### TL;DR - Inviting a new Committer
+
+1. Discuss the proposed committer (and optionally PMC member). (Use
+   [this template](templates/committer-discuss.txt).)
+If the discussion seems to be going positively,
+   call a vote. (Use [this template](templates/committer-vote.txt) for
+   committer, [this template](templates/committer-pmc-vote.txt) for
+   committer and PMC member.)
+1. Close the vote. (Use [this template](templates/committer-results.txt) for a committer vote, or [this template](templates/committer-pmc-results.txt) for a committer + PMC member vote.)
+1. If the result is positive, invite the new committer. (Use [this
```

Comment:
Similar to the comment of @clr-apache above - find and email of the person you want to invite. 

This - sometimes - might be even more challenging than finding a real name. There are valuable contributors who deserve to be invited without writing a single devlist message. The new genertion of developers try to avoid mailing lists and email communication. I had one such example recently, also some of our PMC members almost never communiicate via devlist/mailing lists. 

It might seem trivial, but I think it's importantt to make sure that you have the right person - there are many people with similar names sometimes, especially when they are from different culture, it might be easy to make a mistake. Occasionally you might find two people with the same first and last name and not be sure who is who.

I sometimes struggle with making sure that I have the right email connected with the "real" person (and their "github id") - especially that some privacy aware people choose to anonymise their email address in GitHub. 

Some guidelines for that (not sure if it's worth to add such details in :

* check if the person communicated over devlist 
* check email the person publishes on GitHub
* check emails of commits associated with git commit (might be anonymized by GitHub)
* check with a committer who might know that person
* comment an issue/PR of the person in GitHub to contact you

Reply:
This is discussed, with many of these same details, in the NewCommitter-Discussion section. The Summary/TLDR section is intended to be as brief as possible. Perhaps we could link each bit to the details section below? Other than that, not clear exactly what change is being recommended.

Reply:
Looks good. And yeah, it's likely not going to change much if we be more explicit - and generally, the shorter instructions, the better. you can disregard this one.